pihole-google.txt

```diff
@@ -7619,4 +7619,67 @@ www.tofu.jibemobile.com
 www.tokyo.jibemobile.com
 www.viper.jibemobile.com
 www.watermelon.jibemobile.com
-yellowstone.jibemobile.com
\ No newline at end of file
+yellowstone.jibemobile.com
+# MY SUGGESTIONS
+i0.ytimg.com
+i5.ytimg.com
+youtube-noscript.com
+www.youtube-noscript.com
+goog.le
+domains.googlesyndication.com
+appliedsemantics.com
+ads.googlesyndication.com
+.google-analytics.com
+.googletagservices.com
+.doubleclick.com
+.doubleclick.net
+clients9.google.com
+clients0.google.com
+sassets.vevo.com
+2975c.v.fwmrm.net
+m.v.fwmrm.net
+.fwmrm.net
+ytimg.googleusercontent.com
+html5shiv.googlecode.com
+b3.caspio.com
+lh0.googleusercontent.com
+s.googleusercontent.com
+s0.googleusercontent.com
+s1.googleusercontent.com
+s3.googleusercontent.com
+s4.googleusercontent.com
+s5.googleusercontent.com
+s6.googleusercontent.com
+s7.googleusercontent.com
+s8.googleusercontent.com
+s9.googleusercontent.com
+.ggpht.com
+lh1.ggpht.com
+lh2.ggpht.com
+gp1.googleusercontent.com
+gp2.googleusercontent.com
+gp4.googleusercontent.com
+gp5.googleusercontent.com
+gp6.googleusercontent.com
+gp7.googleusercontent.com
+conversion-pixel.invitemedia.com
+rmcdn.2mdn.net
+rmcdn.f.2mdn.net
+n339.asp-cc.com
+ads.cc-dt.com
+clickserve.cc-dt.com
+creative.cc-dt.com
+clickserve.dartsearch.net
+clickserve.eu.dartsearch.net
+clickserve.uk.dartsearch.net
+motifcdn.doubleclick.net
+motifcdn2.doubleclick.net
+affiliate.2mdn.net
+clickserve.us2.dartsearch.net
+log2.quintelligence.com
+www.destinationurl.com
```

Comment:
```suggestion
```
destinationurl.com is not a Google domain, see https://whois.domaintools.com/destinationurl.com